### PR TITLE
add title attribute to resource badges

### PR DESCRIPTION
**Pull request by nicolethoen**
Add `title` attribute on resource badges. 
Add `sr-only` span with the resource label for screen readers.

### CODE REVIEW

**Comment by coreos-ui** [comment]
Can one of the admins verify this patch?

**Comment by coreos-ui** [comment]
Can one of the admins verify this patch?

**Comment by benjaminapetersen** [comment]
I believe the tag is:

/ok-to-test

File: frontend/public/components/utils/resource-icon.tsx [hunk 1]
```diff
@@ -16,10 +16,14 @@ export const ResourceIcon = (props: ResourceIconProps) => {
   }
   const kindObj = modelFor(kind);
   const kindStr = _.get(kindObj, 'kind', kind);
+  const title = (kindObj && kindObj.label);
   const klass = classNames(`co-m-resource-icon co-m-resource-${kindStr.toLowerCase()}`, className);
   const iconLabel = (kindObj && kindObj.abbr) || kindToAbbr(kindStr);
 
-  const rendered = <span className={klass}>{iconLabel}</span>;
+  const rendered = <React.Fragment>
+    <span className="sr-only">Resource Label</span>
```
Comment (spadgett) [inline frontend/public/components/utils/resource-icon.tsx]:
```jsx
<span className="sr-only">{title}</span>
```

File: frontend/public/components/utils/resource-icon.tsx [hunk 2]
```diff
@@ -16,10 +16,14 @@ export const ResourceIcon = (props: ResourceIconProps) => {
   }
   const kindObj = modelFor(kind);
   const kindStr = _.get(kindObj, 'kind', kind);
+  const title = (kindObj && kindObj.label);
```
Comment (spadgett) [inline frontend/public/components/utils/resource-icon.tsx]: This isn't always going to be defined. Use `kindStr`

**Pull request merged**